bgcval2/functions/applyLandMask.py

```diff
@@ -0,0 +1,153 @@
+#
+# Copyright 2015, Plymouth Marine Laboratory
+#
+# This file is part of the bgc-val library.
+#
+# bgc-val is free software: you can redistribute it and/or modify it
+# under the terms of the Revised Berkeley Software Distribution (BSD) 3-clause license.
+
+# bgc-val is distributed in the hope that it will be useful, but
+# without any warranty; without even the implied warranty of merchantability
+# or fitness for a particular purpose. See the revised BSD license for more details.
+# You should have received a copy of the revised BSD license along with bgc-val.
+# If not, see <http://opensource.org/licenses/BSD-3-Clause>.
+#
+# Address:
+# Plymouth Marine Laboratory
+# Prospect Place, The Hoe
+# Plymouth, PL1 3DH, UK
+#
+# Email:
+# ledm@pml.ac.uk
+#
+"""
+.. module:: applyLandMask
+   :platform: Unix
+   :synopsis: This function applies a land mask. Note that this will not work for p2p code.
+
+.. moduleauthor:: Lee de Mora <ledm@pml.ac.uk>
+
+"""
+
+import numpy as np
+from bgcval2.bgcvaltools.dataset import dataset
+
+
+tmask = {}
+
+
+def loadDataMask(areafile, maskname):
+    global tmask
+    nc = dataset(areafile, 'r')        
+    tmask[(areafile, maskname)] = nc.variables[maskname][:].squeeze()
+    nc.close()
+
+
+def applyLandMask(nc, keys, **kwargs):
+    try:
+        areafile = kwargs['areafile']
+    except:
+        raise AssertionError("applyLandMask:\t Needs an `areafile` kwarg to apply mask")    
```

Comment:
this is a bit dirty - how about you first check if `kwargs['areafile']` exists then use it, without a try/except, if not toss a key error